packages/compartment-mapper/test/scaffold.js

```diff
@@ -420,3 +420,14 @@ export function scaffold(
     });
   }
 }
+
+// Modifies the given object to make it appear to be an ESM module namespace object.
+export const moduleify = obj => {
```

Comment:
`modularize` was right there.

Reply:
`-ify` is conventional! https://github.com/endojs/endo/blob/master/packages/patterns/src/keys/merge-bag-operators.js/#L271